test/support/flow_test_helper.rb

```diff
@@ -151,8 +151,26 @@ def outcome_body
       OutcomePresenter.new(flow.node(state.current_node_name), nil, state).body
     end
 
+    def outcome_title
+      raise "#{state.current_node_name} is not an outcome" unless current_node_type == :outcome
+
+      OutcomePresenter.new(flow.node(state.current_node_name), nil, state).title
+    end
+
     def outcome_body_text
       Nokogiri::HTML::DocumentFragment.parse(outcome_body).text
     end
+
+    def outcome_title_text
+      Nokogiri::HTML::DocumentFragment.parse(outcome_title).text
+    end
```

Comment:
This shouldn't be needed to as we can't have a HTML title (sorry I only learnt that during this review)